scripts/generate/templates/Packing.js

```diff
@@ -0,0 +1,63 @@
+const format = require('../format-lines');
+const { product } = require('../../helpers');
+const { SIZES } = require('./Packing.opts');
+
+// TEMPLATE
+const header = `\
+pragma solidity ^0.8.20;
+
+/**
+ * @dev Helper library packing and unpacking multiple values into bytesXX.
```

Comment:
We need to write some usage examples. This library provides the building blocks for creating your own packed type, I'd like to convey that idea in a simple example in the NatSpec so we don't need to explain through none other comment.